### PR TITLE
Drop support for ansible-core 2.11 and 2.12

### PR DESCRIPTION
**Pull request by felixfontein**
##### SUMMARY
Implement https://github.com/ansible-community/community-topics/discussions/271.

##### ISSUE TYPE
- Docs Pull Request
- Feature Pull Request

##### COMPONENT NAME
various

### CODE REVIEW

**Comment by felixfontein** [comment]
I'm planning to merge this soon after the 8.5.0 release (which will happen probably on Monday).

File: README.md [hunk 1]
```diff
@@ -24,9 +24,7 @@ If you encounter abusive behavior violating the [Ansible Code of Conduct](https:
 
 ## Tested with Ansible
 
-Tested with the current ansible-core 2.11, ansible-core 2.12, ansible-core 2.13, ansible-core 2.14, ansible-core 2.15, ansible-core 2.16 releases and the current development version of ansible-core. Ansible-core versions before 2.11.0 are not supported. This includes all ansible-base 2.10 and Ansible 2.9 releases.
-
-Parts of this collection will not work with ansible-core 2.11 on Python 3.12+.
+Tested with the current ansible-core 2.13, ansible-core 2.14, ansible-core 2.15, ansible-core 2.16 releases and the current development version of ansible-core. Ansible-core versions before 2.11.0 are not supported. This includes all ansible-base 2.10 and Ansible 2.9 releases.
```
Comment (russoz) [inline README.md:27]:
```suggestion
Tested with the current ansible-core 2.13, ansible-core 2.14, ansible-core 2.15, ansible-core 2.16 releases and the current development version of ansible-core. Ansible-core versions before 2.13.0 are not supported. This includes all ansible-base 2.10 and Ansible 2.9 releases.
```

**Review by russoz — COMMENTED** [review]
Just one nit to pick, other than that LGTM

**Review by russoz — APPROVED** [review]

**Comment by felixfontein** [comment]
CI failure is unrelated and should get fixed by #7385.

**Comment by felixfontein** [comment]
@russoz thanks a lot for reviewing this!

**Pull request merged**